### DISCUSSION TITLE
Roadmap and plans for the future

### DISCUSSION THREAD

**Discussion by bernhardh** [filamentphp/filament#385] [General]
**First of all:**
You have done a great job so far and I would really love to use filamentadmin in a real project. I appreciate your work and I totally understand, that this is an open source project and you currently don't earn money with it. 

**What is "my" problem:**
I am planning to start a new project in the next months, where I would love to use filamentadmin. But there are some missing features (sidebar with sublevel menu, json fields, filters with options, etc.) I would need for that. Now I am not sure, if I should start with filament and hope that at least some of the missing features will be implemented within a reasonable time or if I should go with nova or anything else.

**Its a general problem:**
I know, there is a roadmap, but there is no indicator, when or in which order these things will be done. For planing new projects, this would be a very important information, to know if you think feature X will be done in the next days, in 3 months or maybe next year.

**As a suggestion to speed up development:**
Maybe you should use the community a bit more. Define some features or tasks which should be implemented right here in github and allow the community to take care of it and promote this (for example on the roadmap page and maybe the readme.md). Define what you expect and some guidelines and maybe there will someone do it for you. Isn't that how an opensource project should work? You could act just as the project manager or even pick the interesting tasks for yourself and just make sure the other tasks commited code is ok for you. For example I could implement a sublevel navigation.

Just to say it again: Great job. Would love to see filement admin as the "default" admin for laravel.

**Comment by eugenefvdm** [comment]
I feel obliged to reply to this old post as it comes up very high when searching Filament roadmap.

> Maybe you should use the community a bit more. 

Since this post was written there is intense use of the community.

Roadmap for v3:

> https://github.com/filamentphp/filament/discussions/3654

> Would love to see filement admin as the "default" admin for laravel.

That will never happen. Laravel tries to be agnostic when it comes to front-end because it's primarily a back-end framework. Filament uses only Livewire which is only one of two very popular font-end frameworks that Laravel dev's use, the other being VueJS.

VueJS in a way is more mature because it evolved from very mature Javascript frameworks. It's also very popular in enterprise.

My personal preference is Livewire but I've posed the question of Livewire versus VueJS to people like Taylor and the answer that comes back is always ambiguous. I guess being agnostic make perfect sense.
